src/corner_kick/src/containers/Canvas/spritesheetManager.ts

```diff
@@ -114,7 +114,13 @@ export class SpritesheetManager {
     public getTextureByID = (id: number) => {
         // Check that the id is in the correct range
         if (id >= 0 && id < this.textureDictionary.length) {
-            return this.textureDictionary[id];
+            // This is a special ID refering to a white rectangle.
```

Comment:
why?

Reply:
PIXI provides a dimensionless rectangle texture. This will allow us to draw rectangles at any dimension.

Reply:
I'm still a bit confused, why white? I think I'm overthinking this, might be best discussed in person.

Reply:
We are doing a color multiplication operation when tinting. Think of multiplying a color by white as being the same as multiplying a number by one. You get the number/color as a result. So we need to use white textures in order to effectively tint the sprite, any other texture color would result in weird colors when tinting.